modules/assim.sequential/R/Remote_helpers.R

```diff
@@ -94,7 +94,7 @@ Obs.data.prepare.MultiSite <- function(obs.path, site.ids) {
 SDA_remote_launcher <-function(settingPath, 
                                ObsPath,
                                run.bash.args){
-
+  plan(multiprocess)
```

Comment:
Missing the namespace on this function call. Also, I'm not familiar with the `plan` function -- what is this step doing? And what is it acting on? `multiprocess` wasn't an argument to this function.

Reply:
I used a combination of `plan` function form the `future` package  and the ` furrr::future_map` to parallelize some processes. `multiprocess` is also is a 'strategy' defined in the `future` package that automatically checks for the support for multicore to spread the jobs. If it is not supported, then it falls back to multi-session and finally sequential whichever possible to evaluate the jobs.